src/support/stat.c

```diff
@@ -50,8 +50,8 @@ static const char *const __stats_dsrc_desc[] = {
   "cache: pages read into cache by checkpoint requiring cache overflow entries.",
   "cache: pages read into cache requiring cache overflow entries.",
   "cache: pages requested from the cache", "cache: pages seen by eviction walk",
-  "cache: pages written from cache", "cache: pages written requiring in-memory restoration",
```

Comment:
Don't we restore the disk image in the memory after update restore? Is the update restore is same as normal reconciliation?

Reply:
What do you mean by normal reconciliation? Whether to restore the disk image is now controlled by a flag r->leave_dirty, which will be flagged to true if there are uncommitted changes on the page.

Reply:
Yes, I feel it is better to have stats that represent disk image restored evictions.